tests/test_settings_api.py

```diff
@@ -36,7 +36,8 @@ def test_setup_models_viscous_model_settings(new_solver_session) -> None:
     assert viscous_model.model() == "inviscid"
 
 
-@pytest.mark.fluent_version(">=24.1")
+# Failing for 24.1 but passes for 24.2 and 25.1
+@pytest.mark.fluent_version(">=24.2")
```

Comment:
@mkundu1 This test is failing for 24.1 but passing for 24.2 and 25.1 therefore we have updated it's version.